OsmAnd/src/net/osmand/plus/views/layers/NearbyPlacesLayer.java

```diff
@@ -187,9 +162,15 @@ private void drawPoints(List<NearbyPlacePoint> pointsToDraw, QuadRect latLonBoun
 		for (NearbyPlacePoint point : fullObjects) {
 			Bitmap bitmap = point.imageBitmap;
 			if (bitmap != null) {
+				String key = point.photoTitle;
+				NearbyPlaceCacheItem cacheItem = bigBitmapCache.get(key);
+				if (cacheItem == null || cacheItem.isSelected != point.isSelected) {
+					Bitmap bigBitmap = PointImageUtils.createBigBitmap(getApplication(), bitmap, point == selectedObject);
+					cacheItem = new NearbyPlaceCacheItem(bigBitmap, point.isSelected);
+				}
```

Comment:
we only getting & creating items here, without saving it to cache, is it a typo?